lib/Deprecated.js

```diff
@@ -108,10 +118,11 @@ class iPgm {
   }
 
   /**
-   * @description adds a return element to the program XML
-   * @param {string} value
-   * @param {string} type
-   * @param {object} [options]
+   * @deprecated Will be removed in the next major version.
+   * @description Adds a return element to the program XML.
```

Comment:
We should refrain from mentioning XML as much as possible. Something like "Specifies the type of the return value from the service program function."